src/Symfony/Component/AssetMapper/Command/ImportMapInstallCommand.php

```diff
@@ -36,8 +39,36 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $io = new SymfonyStyle($input, $output);
 
-        $downloadedPackages = $this->importMapManager->downloadMissingPackages();
-        $io->success(sprintf('Downloaded %d assets.', \count($downloadedPackages)));
+        $finishedCount = 0;
+        $progressBar = new ProgressBar($output);
+        $progressBar->setFormat('<info>%current%/%max%</info> %bar% %url%');
+        $downloadedPackages = $this->packageDownloader->downloadPackages(function (string $package, string $event, ResponseInterface $response, int $totalPackages) use (&$finishedCount, $progressBar) {
+            $progressBar->setMessage($response->getInfo('url'), 'url');
+            if (0 === $progressBar->getMaxSteps()) {
+                $progressBar->setMaxSteps($totalPackages);
+                $progressBar->start();
+            }
+
+            if ('finished' === $event) {
+                ++$finishedCount;
+                $progressBar->advance();
+            }
+        });
+        $progressBar->finish();
+        $progressBar->clear();
+
+        if (0 === \count($downloadedPackages)) {
```

Comment:
another CS thing I'm not gonna give up with ;P
```suggestion
        if (!$downloadedPackages) {
```